pkgs/tools/typesetting/tex/texlive/tlVersion.nix

```diff
@@ -0,0 +1,15 @@
+{
+  year = "2021";
```

Comment:
First mistake: texlive year and snapshot year are not the same thing. So a better approach is to have attributes mirrors, bin, tlpdb, where bin and tlpdb have their own year/month/day.